platform/docs/docs/migration-guide.md

```diff
@@ -0,0 +1,492 @@
+---
+sidebar_position: 10
+sidebar_label: Migration Guide
+---
+
+# Migration Guide
+
+On this page, we will provide a guide to migrating from OHIF v2 to v3. Please note
```

Comment:
Rephrase suggestion: ```This is the migration guide from OHIF v2 to v3. Please note
that this document is a work in progress and will be updated as we move forward.```

Here you can find general guidance on the why, when and how you should migrate to the next version. This document is not meant to be used as a migration recipe but as a migration overview.